internal/impl/nats/auth/docs.go

```diff
@@ -39,6 +39,7 @@ More details [here](https://docs.nats.io/developing-with-nats/security/creds).`
 func FieldSpec() docs.FieldSpec {
 	return docs.FieldObject("auth", "Optional configuration of NATS authentication parameters.").WithChildren(
 		docs.FieldString("nkey_file", "An optional file containing a NKey seed.", "./seed.nk").Optional(),
+		docs.FieldString("nkey_file_content", "An optional file containing a NKey seed string.", "").Optional(),
```

Comment:
`An optional file` - you don't pass the file to this option right? `An optional NKey seed string` or something like that is more accurate